Sources/GRPCCore/Documentation.docc/Tutorials/Hello-World/Hello-World.tutorial

```diff
@@ -0,0 +1,126 @@
+@Tutorial(time: 10) {
+  @XcodeRequirement(
+    title: "Xcode 16 Beta 5+",
+    destination: "https://developer.apple.com/download/"
```

Comment:
Interesting, I also don't see it locally. It _is_ a requirement though. I wonder if it's not rendered when you generate it with Xcode because you already have an Xcode?

Reply:
🤔